MWStoryGitFile.txt

```diff
@@ -20444,7 +20444,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Nov 29 14:17:28 GMT+0100 2020_12864"MainText""
 "Sun Nov 29 14:17:28 GMT+0100 2020_12867"MainText"You want this, right?"
 "Sun Nov 29 14:17:28 GMT+0100 2020_12870"MainText"You ... the... "
-"Sun Nov 29 14:17:28 GMT+0100 2020_12870"If_story_text"You want to fill me up?"
+"Sun Nov 29 14:17:28 GMT+0100 2020_12870"If_story_text"You want to fill meup?"
```

Comment:
You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<